3-projects/standard-project-example-optional.tf

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "standard_project_folders-optional" {
+  source              = "./modules/folder_environments"
+  parent_folder_id    = google_folder.app.id
+  folder_display_name = "standard_app_example_optional"
+}
+
+module "standard-project-app-optional" {
+  source = "./modules/standard_projects"
+
+  org_id                      = var.organization_id
+  billing_account             = var.billing_account
+  impersonate_service_account = var.terraform_service_account
+
+  nonprod_folder_id = module.standard_project_folders-optional.nonprod_folder_id
+  prod_folder_id    = module.standard_project_folders-optional.prod_folder_id
+
+  # Metadata
+  project_prefix   = "standard-optional"
+  cost_centre      = "cost-centre-1"
+  application_name = "sample-standard-app-optional"
+
+  # Network Setting (Optional)
+  enable_networking            = true
```

Comment:
If we remove the link to the submodules as discussed above, we may need to move this into documentation.